bridge/test/BridgeV3_to_bridgeV4_test.js

```diff
@@ -149,9 +149,7 @@ contract('Bridge Multichain Deploy Check', async function (accounts) {
         amount,
         crossEvent.blockHash,
         crossEvent.transactionHash,
-        crossEvent.logIndex,
-        chains.ETHEREUM_MAIN_NET_CHAIN_ID,
-        chains.HARDHAT_TEST_NET_CHAIN_ID,
+        crossEvent.logIndex
```

Comment:
this was outdated right? I don't see in FederationV2 the chain ids arguments in this event

Reply:
Most of my changes on the tests was to remove outdated values